cmake/JuliaCCompiler.cmake

```diff
@@ -0,0 +1,84 @@
+#
+
+# libc++ is standard on OS X 10.9, but not for earlier releases
+jl_option(USE_LIBCPP "Use libc++" Off)
+# assume we don't have LIBSSP support in our compiler,
+# will enable later if likely true
+set(HAVE_SSP Off)
+
+jl_set_option(USEICC Off)
+jl_set_option(USEGCC Off)
+jl_set_option(USECLANG Off)
+jl_set_option(USEMSVC Off)
```

Comment:
Apparently I missed it (`MSVC`) from the [cmake wiki](http://www.cmake.org/Wiki/CMake_Useful_Variables) probably because it is listed along with many other related ones. In that case I'll remove all of these once they are not needed.

P.S. is `WIN32` set to `TRUE` on 64bit windows as well? The cmake wiki doesn't list anything for WIN64 so I guess it has to? (also too lazy to read cmake source....)